test/test.js

```diff
@@ -491,6 +491,19 @@ suite('Request', function() {
     })
   })
 
+  test('clone request bodies', function() {
+    var r1 = new Request('https://fetch.spec.whatwg.org/', {
+      method: 'post',
+      headers: {'content-type': 'text/plain'},
+      body: 'I work out'
```

Comment:
I've added a test in the latest commit. It may be that this test would never actually since blobs can be re-read without problems normally but it doesn't hurt to ensure that the clone's body `.blob()` works as well and not just `.text()`.

A more thoroughly test would use a big blob that challenged the reader a bit more but I think that's a bit out of scope for this fix.